### PR TITLE
separate print_cookies and save_cookies

### PR DESCRIPTION
**Pull request by yhi00240**
쿠키의 이름은 소문자와 '_'로만 이루어져 있습니다.
하지만 사용자에게 보여줄 때는 learning_rate 가 아니라 Learning Rate 로 보여주어야 하므로,
따로 쿠키 리스트를 선언하여, 이름을 바꿔 출력하도록 했습니다.

### CODE REVIEW

File: practice/views.py [hunk 1]
```diff
@@ -78,11 +78,17 @@ def post(self, request, practice_name):
     @staticmethod
     def check(request, practice_name):
         template = 'practice/training/check.html'
-        print_list = ['model_type', 'activation_function', 'optimizer', 'weight_initialization', 'dropout', 'learning_rate', 'optimization_epoch']
+        print_list = ['Model type', 'Activation Function', 'Optimizer', 'Weight Initialization', 'Dropout', 'Learning Rate', 'Optimization Epoch']
         cookies_list = {}
 
         for i in range(len(print_list)) :
```
Comment (SeoJungHong) [inline practice/views.py]: 파이썬에서 리스트 내부 아이템을 탐색할 때에는 심플하게 `for item in print_list` 이런 식의 문법이 허용되므로 인덱스를 꼭 이용하지 않아도 좋을 것 같아요~

File: practice/views.py [hunk 2]
```diff
@@ -78,11 +78,17 @@ def post(self, request, practice_name):
...
-            cookies_list[print_list[i]]=request.COOKIES.get(print_list[i])
+            cookie_name=''
+            for j in range(len(print_list[i])) :
+                if print_list[i][j]==' ':
+                    cookie_name += '_'
```
Comment (SeoJungHong) [inline practice/views.py]: 현재 스트링에 빈칸이 있으면 `_`로 바꾸고 전체 소문자를 적용하는 것은 python에서 제공하는 함수들을 사용하는게 더 알아보기 좋을 것 같습니다. `replace(old, new)` 라는 함수가 있는데 스트링에 old패턴의 문자가 있으면 new로 변경해줍니다. 따라서 위의 코드는
```python
for item in print_list:
    item.replace(' ', '_').lower()
```
 이런 식으로 한번에 적용해서 간단히 바꿀 수 있을 것 같아요.

Comment (yhi00240) [inline practice/views.py]: 오 한번에 바꾸면 되면 되는군요! 넵 수정할게요~
